From 29100bc1e2e46593a3ea7956e2a51f008ed8f747 Mon Sep 17 00:00:00 2001
From: sjorge <sjorge@blackdot.be>
Date: Fri, 2 Jun 2017 09:23:38 +0000
Subject: [PATCH] joyent/smartos-live#724 Cannot build smartos-live on
 restricted networks Reviewed by: Robert Mustacchi <rm@joyent.com> Reviewed
 by: Cody Peter Mello <cody.mello@joyent.com> Approved by: Cody Peter Mello
 <cody.mello@joyent.com>

---
 sample.configure.smartos | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/sample.configure.smartos b/sample.configure.smartos
index 879b6821..a0c04922 100644
--- a/sample.configure.smartos
+++ b/sample.configure.smartos
@@ -1,12 +1,12 @@
 mkdir -p projects/local
 (cd projects/local >/dev/null && [[ -d ur-agent ]] \
-    || git clone git://github.com/joyent/sdc-ur-agent.git ur-agent)
+    || git clone https://github.com/joyent/sdc-ur-agent.git ur-agent)
 (cd projects/local >/dev/null && [[ -d kvm ]] \
-    || git clone git://github.com/joyent/illumos-kvm.git kvm)
+    || git clone https://github.com/joyent/illumos-kvm.git kvm)
 (cd projects/local >/dev/null && [[ -d kvm-cmd ]] \
-    || git clone git://github.com/joyent/illumos-kvm-cmd.git kvm-cmd)
+    || git clone https://github.com/joyent/illumos-kvm-cmd.git kvm-cmd)
 (cd projects/local >/dev/null && [[ -d mdata-client ]] \
-    || git clone git://github.com/joyent/mdata-client.git mdata-client)
+    || git clone https://github.com/joyent/mdata-client.git mdata-client)
 
 # default branch list if there isn't one
 if [[ ! -f configure-branches ]]; then
@@ -27,7 +27,7 @@ RELEASE_VER="joyent_147"
 SUNW_SPRO12_URL="https://download.joyent.com/pub/build/SunStudio.tar.bz2"
 ON_CLOSED_BINS_URL="https://download.joyent.com/pub/build/illumos/on-closed-bins.i386.tar.bz2"
 ON_CLOSED_BINS_ND_URL="https://download.joyent.com/pub/build/illumos/on-closed-bins-nd.i386.tar.bz2"
-GET_ILLUMOS="git clone git://github.com/joyent/illumos-joyent.git illumos"
-GET_ILLUMOS_EXTRA="git clone git://github.com/joyent/illumos-extra.git illumos-extra"
+GET_ILLUMOS="git clone https://github.com/joyent/illumos-joyent.git illumos"
+GET_ILLUMOS_EXTRA="git clone https://github.com/joyent/illumos-extra.git illumos-extra"
 ILLUMOS_ADJUNCT_TARBALL_URL="https://download.joyent.com/pub/build/adjuncts/"
 OVERLAYS="generic"
-- 
2.21.0

